src/PlaintextApplicator.cpp

```diff
@@ -272,7 +272,7 @@ void PlaintextApplicator::runGrammarOnText(std::istream& input, std::ostream& ou
 					lSWindow->text += &line[0];
 				}
 				else {
-					u_fprintf(output, "%S", &line[0]);
+					printPlainTextLine(&line[0], output);
```

Comment:
Why are there any other changes in this file? Seems like a patch gone wrong.

Reply:
I blindly put printPlainTextLine and printStreamCommand inside all of the runGrammarOnText() functions anywhere that they were printing directly to output. I haven't gone back to actually evaluate whether they're needed there yet.